notebooks/WFC3/ir_ima_visualization/IR_IMA_Visualization_with_an_Example_of_Time_Variable_Background.ipynb

```diff
@@ -96,7 +96,6 @@
     "- *matplotlib.pyplot* for plotting data\n",
```

Comment:
<p>I noticed that the <code>alt text</code> for the image in the document is currently set to <code>drawing</code> For improved accessibility and to aid understanding for those who might not be able to see the image, it would be beneficial to provide more descriptive alt text.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>